skore/src/skore/sklearn/_plot/metrics/metrics_summary_display.py

```diff
@@ -9,19 +13,166 @@ class MetricsSummaryDisplay(HelpDisplayMixin, StyleDisplayMixin):
 
-    @StyleDisplayMixin.style_plot
-    def plot(self):
-        raise NotImplementedError
+    def _plot_comparison_estimator(self, x, y):
+        self.figure_, self.ax_ = plt.subplots()
+
+        x_label = self._SCORE_OR_LOSS_INFO.get(x, {}).get("name", x)
+        y_label = self._SCORE_OR_LOSS_INFO.get(y, {}).get("name", y)
+
+        # Check that the metrics are in the report
+        # If the metric is not in the report, help the user by suggesting
+        # supported metrics
+        reverse_score_info = {
+            value["name"]: key for key, value in self._SCORE_OR_LOSS_INFO.items()
+        }
+        index = self.summarize_data.index
+        if isinstance(index, pd.MultiIndex):
+            available_columns = index.get_level_values(0).to_list()
+        else:
+            available_columns = index.tolist()
+        supported_metrics = [
+            reverse_score_info.get(col, col) for col in available_columns
+        ]
+        if x not in supported_metrics:
+            raise ValueError(
+                f"Performance metric {x} not found in the report. "
+                f"Supported metrics are: {supported_metrics}."
+            )
+        if y not in supported_metrics:
+            raise ValueError(
+                f"Performance metric {y} not found in the report. "
+                f"Supported metrics are: {supported_metrics}."
+            )
+
+        x_data = self.summarize_data.loc[x_label]
+        y_data = self.summarize_data.loc[y_label]
+        if len(x_data.shape) > 1 and x_data.shape[0] > 1:
+            # case where we have multiIndex, and the metric is not a single value
+            raise ValueError(
+                "The perf metric x requires to add a positive label parameter."
+            )
+        elif len(x_data.shape) > 1 and x_data.shape[0] == 1:
+            # case where we have multiIndex, but the metric is not affected by the
+            # pos_label
+            x_data = x_data.squeeze()
+        if len(y_data.shape) > 1 and y_data.shape[0] > 1:
+            raise ValueError(
+                "The perf metric y requires to add a positive label parameter."
+            )
+        elif len(y_data.shape) > 1 and y_data.shape[0] == 1:
+            y_data = y_data.squeeze()
+
+        # Make it clear in the axis labels that we are using the train set
+        if x == "fit_time" and self.data_source != "train":
+            x_label_text = x_label + " on train set"
+        else:
+            x_label_text = x_label
+        if y == "fit_time" and self.data_source != "train":
+            y_label_text = y_label + " on train set"
+        else:
+            y_label_text = y_label
+
+        title = f"{x_label} vs {y_label}"
+        if self.data_source is not None:
+            title += f" on {self.data_source} set"
+
+        self.ax_.scatter(x=x_data, y=y_data)
+        self.ax_.set_title(title)
+        self.ax_.set_xlabel(x_label_text)
+        self.ax_.set_ylabel(y_label_text)
```

Comment:
A more compact way to `set` stuff in matplotlib.

```suggestion
        self.ax_.set(title=title, xlabel=x_label_text, y_label=y_label_text)
```